i18n-helpers/src/normalize.rs

```diff
@@ -566,4 +576,12 @@ mod tests {
             ],
         );
     }
+
+    #[test]
+    fn test_normalize_pot_creation_date() {
```

Comment:
Please also add a test for the case where the date cannot be parsed.

Reply:
Done!